lib/UniPublish.js

```diff
@@ -120,7 +120,19 @@ export default UniCollection => {
             --this._uniDocCounts[collectionName][id];
             if (!this._uniDocCounts[collectionName][id]) {
                 delete this._uniDocCounts[collectionName][id];
-                _stopObserveHandlesAndCleanUp.call(this, collectionName, id);
+
```

Comment:
`_stopObserveHandlesAndCleanUp` won't  be called if there are no mappings, this is intended behavior?